Sources/Sentry/Public/SentrySpanContext.h

```diff
@@ -1,12 +1,13 @@
 #import "SentryDefines.h"
 #import "SentrySerializable.h"
+#import "SentrySpanStatus.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @class SentryId, SentrySpanId;
 
-NS_SWIFT_NAME(SpanContext)
-@interface SentrySpanContext : NSObject
+NS_SWIFT_NAME(SentrySpanContext)
```

Comment:
nope, because this could conflict with other 3rd party libs, I brought up the same topic yesterday, better to play safe

Reply:
We have namespaces in Swift, so what is the problem?

Reply:
The Swift name is not a problem. I was keeping the same name for consistence, but I can chance it.

Reply:
ah ok its just about the Swift ones, my bad

Reply:
Not a problem in Swift, I was lost in naming conventions, fixing it.

Reply:
Im kind lost now. Some classes in Java (at least SentryTransaction) uses the Sentry prefix, but .NET doesn't. Should I remove the Sentry prefix in all Performances classes in Swift

Reply:
@brustolin I don't think there was any guidance about at least afaik, so I believe folks just created it, maybe adding `Sentry` as prefix if could cause import conflicts.

as discussed before, ObjC must add the Sentry prefix, Swift has namespaces so it's not needed, well Java also does and it adds some Prefix in some classes because it could conflict even with the JDK classes.

I'd let @philipphofmann chose the approach here.

I don't think we can align this now in Java and .NET as they are GA already, maybe @bruno-garcia has opinions

Reply:
As discussed in a call with @bruno-garcia, @marandaneto, and @philipphofmann, we don't prefix in Swift, as this is the case for other classes like Event, Scope, UserFeedback, Attachment, Client, Hub, ...